lua/codeium/api.lua

```diff
@@ -381,28 +381,37 @@ function Server:new()
 				return
 			end
 			local ports = vim.fn.json_decode(body)
-			os.execute(
-				"xdg-open 'http://127.0.0.1:"
-					.. ports.chatClientPort
-					.. "?api_key="
-					.. api_key
-					.. "&has_enterprise_extension="
-					.. (config.options.enterprise_mode and "true" or "false")
-					.. "&web_server_url=ws://127.0.0.1:"
-					.. ports.chatWebServerPort
-					.. "&ide_name=neovim"
-					.. "&ide_version="
-					.. versions.nvim
-					.. "&app_name=codeium.nvim"
-					.. "&extension_name=codeium.nvim"
-					.. "&extension_version="
-					.. versions.extension
-					.. "&ide_telemetry_enabled=true"
-					.. "&has_index_service="
-					.. (config.options.enable_index_service and "true" or "false")
-					.. "&locale=en_US"
-					.. "'"
-			)
+			local url = "http://127.0.0.1:"
+				.. ports.chatClientPort
+				.. "?api_key="
+				.. api_key
+				.. "&has_enterprise_extension="
+				.. (config.options.enterprise_mode and "true" or "false")
+				.. "&web_server_url=ws://127.0.0.1:"
+				.. ports.chatWebServerPort
+				.. "&ide_name=neovim"
+				.. "&ide_version="
+				.. versions.nvim
+				.. "&app_name=codeium.nvim"
+				.. "&extension_name=codeium.nvim"
+				.. "&extension_version="
+				.. versions.extension
+				.. "&ide_telemetry_enabled=true"
+				.. "&has_index_service="
+				.. (config.options.enable_index_service and "true" or "false")
+				.. "&locale=en_US"
+
+			-- cross-platform solution to open the web app
+			local os_info = io.get_system_info()
+			if os_info.os == "linux" then
+				os.execute("xdg-open '" .. url .. "'")
+			elseif os_info.os == "macos" then
+				os.execute("open '" .. url .. "'")
+			elseif os_info.os == "windows" then
+				os.execute("start " .. url)
```

Comment:
Should the Windows URL also be escaped?

Reply:
> Should the Windows URL also be escaped?

No, the window on browser can opening. 
@DajmondFM are you authenticated? and could you please text the link you get after entering the command :Codeium Chat

Reply:
I think i'm authenticated, because autocompletion works. `http://127.0.0.1:51589/?api_key=15be80b2-8251-4c9c-8056-16ea4abf4f81`